resources/views/pdf.blade.php

```diff
@@ -129,7 +134,13 @@
 
 <body>
     @if ($numbering !== false)
-        <footer></footer>
+        <footer>
+            @if (in_array('date', $options))
+                <div class="date">
+                    Printed {{ date('Y-m-d') }}
```

Comment:
This pushes the page number down. I'd prefer that it didn't do that, but my CSS skills are ... non-existent!

Reply:
also it's not great to have the date be align-right, since that will be on the inside edge of left-side pages when booklet printing

is there a need to have it repeated on every page? we recommend prepending a custom cover page. the san jose intergroup puts the date on their cover - [this is the source document they use](https://docs.google.com/document/d/11NO2bOD7OLq9gy7zTG_0JED1Z2l570mj9g7L61Yc77Y/edit) for that

Reply:
We're experimenting with printing a one page folded cover that can be distributed without the meeting information. For that reason, the cover won't have a date, but will have a QR code to the website, other pertinent contact information, and a place for people to add names/phone numbers.

Then we'll print fewer of these meeting listings which can be easily inserted into the cover. So the meeting listing itself should have the date. It doesn't need to be on every page. It would be fine if it was just on the legend page.

Reply:
How about a header textarea and footer textare for the legend page. That way I could just put whatever text I wanted on it, such as the Printed date?